rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -47,6 +47,17 @@ def fromcstring3(s):
 	except AttributeError as e:
 		raise
 
+class ReturnCode:
+	ok = 0
+	no_data = 11
+
+class DdsError(Exception):
+	pass
+
+def check_retcode(retcode):
+	if retcode != ReturnCode.ok and retcode != ReturnCode.no_data:
```

Comment:
depending on whether or not you think this list of `if` is going to grow, we could modify it so that it gets all of the values within the class ReturnCode (example: https://stackoverflow.com/questions/1398022/looping-over-all-member-variables-of-a-class-in-python)

Reply:
I don't think it will grow. The only two non-exception retcodes are ok and no-data. Maybe timeout could eventually be added, but nothing more.